server/build.gradle

```
@@ -186,10 +187,24 @@ jib {
     }
 }
 
+jacoco {
+    toolVersion = "0.8.12" // Updated to latest version
```

The comment seems a bit odd since we've not had a jacoco version before that is being updated?

```suggestion
    toolVersion = "0.8.12" 
```

---

yes, double on that. It might be misleading to have "Updated".